bindings/python/eblob_python.cpp

```diff
@@ -75,18 +75,17 @@ struct eblob_py_iterator : eblob_iterate_control, boost::python::wrapper<eblob_i
 	static int iterator(struct eblob_disk_control *dc, struct eblob_ram_control *rc __attribute__((unused)),
 			    int fd, uint64_t data_offset, void *priv, void *thread_priv __attribute__((unused)))
 	{
-		std::unique_ptr<char> p(new char[dc->data_size]);
+		std::string p(dc->data_size, 0);
 
-		::ssize_t bytes = pread(fd, p.get(), dc->data_size, data_offset);
-		if (bytes != static_cast< ::ssize_t>(dc->data_size))
```

Comment:
You have to leave this space because all compilers except trusty's are fail to compile it:
```bash
/eblob/bindings/python/eblob_python.cpp: In static member function 'static int eblob_py_iterator::iterator(eblob_disk_control*, eblob_ram_control*, int, uint64_t, void*, void*)':
/eblob/bindings/python/eblob_python.cpp:81:27: error: expected '<' before '<:' token
/eblob/bindings/python/eblob_python.cpp:81:27: error: expected type-specifier before '<:' token
/eblob/bindings/python/eblob_python.cpp:81:27: error: expected '>' before '<:' token
/eblob/bindings/python/eblob_python.cpp:81:27: error: expected '(' before '<:' token
/eblob/bindings/python/eblob_python.cpp:81:29: error: expected identifier before ':' token
/eblob/bindings/python/eblob_python.cpp:82:4: error: expected ']' before 'return'
/eblob/bindings/python/eblob_python.cpp: In lambda function:
/eblob/bindings/python/eblob_python.cpp:82:4: error: expected '{' before 'return'
/eblob/bindings/python/eblob_python.cpp: In static member function 'static int eblob_py_iterator::iterator(eblob_disk_control*, eblob_ram_control*, int, uint64_t, void*, void*)':
/eblob/bindings/python/eblob_python.cpp:82:4: error: expected ')' before 'return'
/eblob/bindings/python/eblob_python.cpp:82:4: error: expected ')' before 'return'
```